common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -141,14 +138,18 @@ open class MusicService : MediaBrowserServiceCompat() {
          */
         sessionToken = mediaSession.sessionToken
 
-        // Because ExoPlayer will manage the MediaSession, add the service as a callback for
-        // state changes.
-        mediaController = MediaControllerCompat(this, mediaSession).also {
-            it.registerCallback(MediaControllerCallback())
-        }
-
-        notificationBuilder = NotificationBuilder(this)
-        notificationManager = NotificationManagerCompat.from(this)
+        /**
```

Comment:
I inherited this style so didn't want to break from convention.

Reply:
Ah, so the reason I used KDoc in many of the comments was so I could link to other methods/classes with `[Class]` or `[method]`. Since there aren't any links, I concur with Ben's thoughts.